scripts/check-RN-link-perms.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Checks if a release notes file is updated. If true, searches the release notes file for links to bugs that are behind a login and fails the build.
+# We should not include internal bugs in our release notes.
+
+# Get the repo path and files changed
+REPO_PATH="$(git rev-parse --show-toplevel)"
+FILES=$(git diff --name-only HEAD~1 HEAD --diff-filter=d "*.adoc" ':(exclude)_unused_topics/*')
+
+# Function to check links in updated release notes
+check_rn_links () {
+    
+    #Extract the release notes file from all the updated files
+    while IFS= read -r line; do
+        if echo "$line" | grep -q "release"; then
```

Comment:
This seems brittle - there are lots of files with "release" in the filename.

Reply:
Good point. Perhaps "release-notes" would cover it